core/src/main/java/com/graphhopper/routing/querygraph/QueryGraph.java

```diff
@@ -340,4 +340,16 @@ private UnsupportedOperationException exc() {
     public Graph getMainGraph() {
         return mainGraph;
     }
+
+    public int shiftEdgeId(int edgeId) {
```

Comment:
Ok yes this needs a better name and maybe more documentation, but we can consider this work in progress and it will be removed again soon anyway :)